protocols/Newport_scientific/LCMS_bioassay_part1.ot2.py

```diff
@@ -0,0 +1,77 @@
+from opentrons import labware, instruments
+
+# customization
+sample_num = 96
+reagent_row_1 = 6
+
+incubation_plate = labware.load('96-PCR-tall', 1)
+incubation_plate2 = labware.load('96-PCR-tall', 2)
+sample_plate = labware.load('96-PCR-tall', 5)
+reagent_rack = labware.load('tube-rack-2ml', 4)
+tiprack = labware.load('tiprack-200ul', 7)
+tiprack2 = labware.load('tiprack-200ul', 8)
+
+buffer_loc = reagent_rack.wells('A1')
+enzyme_loc = reagent_rack.wells('A2')
+is_loc = reagent_rack.wells('A3')
+water_loc = reagent_rack.wells('A4')
+meoh_loc = reagent_rack.wells('A5')
+gluc_std_loc = reagent_rack.wells('A6')
+std = reagent_rack.wells('B1', 'B2', 'B3', 'B4', 'B5', 'B6', 'C1', 'C2',
+                         'C3', 'C4')
+naive_loc = reagent_rack.wells('C5')
+QC = reagent_rack.wells('C6', 'D1', 'D2')
+
+sample_dest = []
+sample_source = []
+well_count = 0
+if sample_num <= 72:
+    sample_dest_plate = incubation_plate.rows[2:]
+else:
+    sample_dest_plate = incubation_plate2.rows()
+
+for source_row, dest_row in zip(sample_plate.rows(), sample_dest_plate):
+    for source_well, dest_well in zip(source_row, dest_row):
+        if well_count < sample_num:
+            sample_dest.append(dest_well)
```

Comment:
use comments (esp at top of file) because I don't know what is the difference between part 1 and part 2 if I'm just receiving these/opening them up.